src/node_crypto.cc

```diff
@@ -741,6 +743,64 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void emitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+#if 0
+  Local<Object> process = env->process_object();
+  Local<v8::Value> f = process->Get(
+    FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
+  };
+  f.As<v8::Function>()->Call(process, arraysize(args), args);
+
+#else
+  Local<Object> process = env->process_object();
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
```

Comment:
Oh, and you could write this as:
```c++
Local<Value> arg = node::OneByteString(env->isolate(), warning);
MakeCallback(env, process, "emitWarning", 1, &arg);
```
It's a little more succinct but whatever you prefer.